test/openAPI/features/oidc_well_known_jwks.feature

```diff
@@ -1,15 +1,15 @@
 @method=GET @endpoint=/.well-known/jwks.json
 Feature: Endpoint to retrieve all the public keys of the IdP server
 
-  Endpoint under discussion. Endpoint to fetch all the public keys of the IdP server.
+  Endpoint to fetch all the public keys of the IdP server.
   Returns public key set in the JWKS format.
 
   @smoke @unit @positive
   Scenario: Successfully retrieves the IdP server's public keys
     Given Wants to retrieve the IdP server's public keys
     When GET request to retrieve all the public keys of the IdP server is sent
     Then The response is received
-    And The response should be returned in a timely manner
+    And The response should be returned in a timely manner 15000ms
     And The response should have status 200
     And The response header content-type should be "application/json; charset=utf-8"
     And The response should match json schema
```

Comment:
The test titles should be more specific because they will repeated with another tests. Just add GET /.well-known/jwks.json to every, example:
`Then The GET .well-known/jwks.json response is received`